translations/binaries/slps_strings_locations.json

```
@@ -0,0 +1,2636 @@
+{
+    "SLPS": [
+        {
+            "offset": 1714264,
+            "string": "カルビオラ神殿・神の眼前戦[END]",
+            "original_bytes_length": 27,
+            "available_bytes_size": 31,
+            "translation": "칼비올라 신전・신의 눈앞 전투[END]"
```

https://github.com/ToD-DC-Kor/kor-patch/pull/445/files#r1773419721 에 적어주신 의견에 따르면, 여기도 까르비올라가 되어야겠군요

```suggestion
            "translation": "까르비올라 신전・신의 눈앞 전투[END]"
```